packages/content/src/types.ts

```diff
@@ -0,0 +1,137 @@
+  
+  // Additional metadata from custom/formal elements
+  metadata?: {
+      // Unique or additional formal metadata that will be displayed in sidebar
+  };
+
+  // Hub configuration metadata
+  actionLinks?: IActionLink[]; // item.properties.links
+  actionConfigs?: Object; // item.properties.actions - enable/disable standard actions like `createWebmap` or `createStorymap`
+
+  metrics?: { // Set visibility for telemetry metrics. Nested object future-proofing, but could flatten.
+      visibility: 'private' | 'org' | 'updateGroups' | 'public'  ; // TODO: should we share with specific groups?
+  };
+
+  contentDisplay?: 'thumbnail' | 'map'; // [Future] View configuration options such as cartography, charts, table, etc.
+  source: IHubSocial; // [Future] each of these has common metadata like `title`, `thumbnail` , and `link`
+  
+  // Content specific values. Combination of relevant item.data, layer info, enrichments, configuration settings
+  // could use values instead - which is common within item.data.values
+  attributes?: {
+      // Dataset, e.g.
+      fields?: [{name:string, type:string, statistics:Array<any>}];
+      recordNumber?: number;
+      // Map, e.g.
+      layers?: [];
+      basemap?: string;
+      // Document
+      format?: 'link' | 'PDF' | 'MS Word' | 'MS Excel' | 'Text';
+      // Initiative
+      stage?: string;
+      followersNumber?: number;
+      coreTeamId?: string; 
+      contentTeamId?: string;
+      supportingTeamsNumber?: number;
+      // Event
+      sponsors?: Array<any>;
+      attendeesNumber?: number;
+      limitNumber?: number;
+      startDate?: Date;
+      endDate?: Date;
+      videoUrl?: string;
+  };
+}
+
+// Common interface to people networks: Teams, Orgs, Members
+export interface IHubSocial {
+  id: string;
+  name: string;
+  thumbnailUrl?: string;
+  region: string; // user.region - US/Europe/etc.
+  culture: string; // e.g. user.culture
+  createdDate: Date;
+  updatedDate: Date;
+  url: string;
+}
+export interface IHubOrg extends IHubSocial { }
+
+export interface IHubTeam extends IHubSocial  { }
+
+// Simple user info - more could be added/cached
+export interface IHubMember extends IHubSocial {
+  username: string; // user.username
+  thumbnailUrl?: string; // user.thumbnail
+}
+
+// title, description, and optional link to license item with more info
+export interface IHubLicense {
+  id: string; // Licenses may be Items, so provide reference for links
+  title: string;
+  description: string;
+  link: string;
+  thumbnailUrl: string;
+  capabilities: {
+    reuse: boolean;
+    commercial: boolean;
+    derivatives: boolean;
+    attribution: boolean;
+    osm: boolean;
+  }
+}
+
+export interface IHubGeography {
+  title:string;
+  coverage?: 'global' | 'regional' | 'local'; // enrichment
+  geometry?: string; // serialized JSON, or should this refer to /resources/boundary.json
```

Comment:
To be useful, it should be a geometry - question is: geojson? or jsapi geometry?

Reply:
Yeah, is there a reason this would be serialized to a string?

Either way, it will be [ArcGIS Geometry](https://developers.arcgis.com/documentation/common-data-types/geometry-objects.htm)

Reply:
Yes, I wasn't aware of [ArcGIS IGeometry](https://github.com/Esri/arcgis-rest-js/blob/13bbea4ee995c1a7ba6936ea4ad16c3ff887927f/packages/arcgis-rest-types/src/geometry.ts) - but that probably is the right pattern.